generators/bootstrap-application-base/generator.ts

```diff
@@ -41,20 +45,21 @@ import { lookupCommandsConfigs } from '../../lib/command/lookup-commands-configs
 import { loadCommandConfigsIntoApplication, loadCommandConfigsKeysIntoTemplatesContext } from '../../lib/command/load.js';
 import { getConfigWithDefaults } from '../../lib/jhipster/default-application-options.js';
 import { isWin32 } from '../base-core/support/index.ts';
-import { removeFieldsWithNullishValues } from '../../lib/utils/index.js';
+import { mutateData, removeFieldsWithNullishValues } from '../../lib/utils/index.js';
 import {
   convertFieldBlobType,
   getBlobContentType,
   isFieldBinaryType,
   isFieldBlobType,
 } from '../base-application/internal/types/field-types.ts';
-import type { EntityAll } from '../base-application/entity-all.js';
 import { upperFirst } from '../../lib/jdl/core/utils/string-utils.js';
 import { baseNameProperties } from '../project-name/support/index.js';
 import { createAuthorityEntity, createUserEntity, createUserManagementEntity } from './utils.js';
 import { exportJDLTransform, importJDLTransform } from './support/index.js';
 
-export default class BootstrapApplicationBase extends BaseApplicationGenerator {
+type Entity = CommonEntity & EntityAll;
```

Comment:
Shouldn't EntityAll extends CommonEntity?

Reply:
Matter of preference: on my side I would prefer to modify types.d later on as there's less code